test/integration_test/basic_test.go

```diff
@@ -307,6 +308,61 @@ func BasicInstall(tc *types.TestCase) func(*testing.T) {
 	}
 }
 
+func TestBasicInstall(t *testing.T) {
+	testCase := &types.TestCase{
+		TestName: "BasicInstall",
+		TestSpec: func(t *testing.T) interface{} {
+			objects, err := ci_utils.ParseSpecs("storagecluster/storagecluster-with-all-components.yaml")
+			require.NoError(t, err)
+			cluster, ok := objects[0].(*corev1.StorageCluster)
+			require.True(t, ok)
+			cluster.Name = "test-basic-stc"
+			cluster.Namespace = ci_utils.PxNamespace
+			return cluster
+		},
+		TestFunc: BasicInstallAetosDemo,
+	}
+	BasicInstallAetosDemo(testCase)(t)
+}
+
+func BasicInstallAetosDemo(tc *types.TestCase) func(*testing.T) {
+	return func(t *testing.T) {
+		testSpec := tc.TestSpec(t)
+		cluster, ok := testSpec.(*corev1.StorageCluster)
+		require.True(t, ok)
+		dash.TestCaseBegin("BasicInstall", "Deploying and validating StorageCluster", "", nil)
+
+		// Construct StorageCluster
+		aetos.LogInfoD("Constructing StorageCluster %s", cluster.Name)
+		err := ci_utils.ConstructStorageCluster(cluster, ci_utils.PxSpecGenURL, ci_utils.PxSpecImages)
+		dash.VerifySafely(err == nil, true,
+			fmt.Sprintf("Constructed StorageCluster %s? %v", cluster.Name, err))
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		// Deploy PX and validate
+		aetos.LogInfoD("Deploying & Validating StorageCluster %s", cluster.Name)
+		_, err = ci_utils.DeployAndValidateStorageClusterAetosDemo(cluster, ci_utils.PxSpecImages)
+		dash.VerifySafely(err == nil, true,
```

Comment:
How about? So, we need not include an error message in the description.

```go
dash.VerifySafely(err, nil, fmt.Sprintf("Deployed and validated StorageCluster %s?", cluster.Name))
```

Reply:
We would want the error to be logged to Aétos.